Schemas_draft-03/Collaboration/DocumentReference/document_reference_POST.json

```diff
@@ -2,6 +2,9 @@
     "title": "document_reference_POST",
     "type": "object",
     "properties": {
+        "guid": {
```

Comment:
Unless I don't understand the purpose of document reference, this makes no sense to include the "guid" here. Document reference should refer to an _existing_ document in the project.

Reply:
A document reference could be either a link to an existing document or to an external url. For external URL's I think it makes sence to include the guid.